package.json

```diff
@@ -24,11 +24,8 @@
     "chrome",
     "browser"
   ],
-  "author": "Dmytro TOLSTYI <dmytro.tolstyi@gmail.com>",
+  "author": "2do2go team <dev.2do2go@gmail.com>",
   "license": "MIT",
-  "xo": {
```

Comment:
My suggestion is to return spaces back for compatibility reasons. Currently it's nearly impossible to see difference (pull/push change) with [upstream project](https://github.com/dtolstyi/node-chromium).

Reply:
Ok, but using spaces is sooooo wrong, imo)